### DISCUSSION TITLE
Is it planned in PrimeNG / Primevue / PrimeReact to have a "Planning Calendar" ?

### DISCUSSION THREAD

**Discussion by Horos34** [primefaces/community#1122] [PrimeNG] [answered]
Hello guys,

I have been working on a personnal project for a while and i want to add a calendar to plan my days / weeks / months / years. 
I mean by that a sort of Angular Calendar but with Prime. I want to stay close to PrimeFlex style and have the freedom of customizing my calendar, events, meetings etc... 

I would like to know if PrimeFaces has planned to add this component ? Or if not, could it be achievable or not ?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Unfortunately, we don't have such a component yet, but you can utilize any other calendar library with PrimeNG.

**Reply by Horos34** [reply]
Hi,
thank you for your answer. 

Could I suggest you guys to add this component ?
